samples/app/MeshAssimp.cpp

```diff
@@ -458,29 +506,32 @@ void MeshAssimp::addFromFile(const Path& path,
         std::vector<uint32_t> indices;
         std::vector<half4> positions;
         std::vector<short4> tangents;
-        std::vector<half2> texCoords;
+        std::vector<half2> texCoords0;
```

Comment:
We should do what @prideout did for `filamesh` here and use `short2` to store either half floats or snorm16s depending on the span of the UVs. UVs between -1 and 1 should use snorm16 for better precision. This can be done in another PR.

Reply:
Yup, you can search for "snormuvs" in `filamesh/src/main.cpp` to see how I did this.